region/norfair/east/Frog Speedway.json

```diff
@@ -819,11 +819,20 @@
       ]
     },
     {
-      "id": 35,
       "link": [2, 2],
-      "name": "G-Mode Setup - Get Hit By Beetom",
+      "name": "G-Mode Setup - Get Hit By Beetom (Speedless Speedway)",
```

Comment:
GMode setup with speed is just gone then?

Reply:
Huh, very confused. I'll fix it